docs/design/schema-compatibility.md

```diff
@@ -0,0 +1,63 @@
+# Support for forward and backward compatibility in DCL schemes
+
+Schema changes can cover a wide range of modifications with varying impacts on application compatibility and data integrity. Below are use cases with strategies to manage schema changes and ensure compatibility.
+
+## Strategy for Compatible Changes
+
+For changes that are backward-compatible, such as adding optional fields or extending enumerations:
+
+**Strategy steps:**
+
+- One time actions:
+  - Add a version field to DCL schema to track the schema version.
+- For each update:
+  - Update the schema.
+  - Probably need to update transactions and queries.
```

Comment:
Let's also add the following:
- DCL doesn't fulfill the Schema version automatically
- It will be up to the transaction submitter (Vendor) to specify a correct Schema version
- If Schema Version is not set - then the initial version (version 0 or 1) is assumed
- It will be up to the client application to process the Schema version

Reply:
Done